src/app/dev/DevToys.Api/Tool/GUI/Components/IUIDiffListInput.cs

```diff
@@ -0,0 +1,115 @@
+﻿namespace DevToys.Api;
+
+/// <summary>
+/// A component that can be used to display compared lists.
+/// </summary>
+public interface IUIDiffListInput : IUIMultiLineTextInput
```

Comment:
I'm not sure what is the interest of adding this public API to the SDK because:
1. It seems to be something specifically needed for the tool you're creating. I can not think of another scenario where it could be reused.
2. The point of adding an API like this would be to take advantage of the `IsCaseSensitiveComparisonMode` and `ComparaisonMode` properties. I see that your tool set them, but I don't see them being used. Also, setting these properties don't seem to have an effect on the UI element or the tool behavior (based on your implementation).

Unless I'm missing an important point here, I'd suggest to remove this change.

Reply:
@veler I set the "ComparisonMode" in "ListCompareGuiTool.cs". That's why I've created the component here. 
Do you see a best way to do it ?

Reply:
That is the problem in fact 😊 Your code **_set_** `ComparaisonMode` property in `ListCompareGuiTool`, but never **_get_**. Therefore, there's no "usage" of its value, so it can be removed. Similarly, there's a `IsCaseSensitiveComparisonMode` property that is not set, neither read. Therefore, there is in my opinion no point at having `IUIDiffListInput`.

Using `IUIMultiLineTextInput` instead of `IUIDiffListInput` would make the List Comparer tool working exactly the same way it does at the moment, minus removing `.CompareModeAInterB()`, `.CompareModeBOnly()` and `.CompareModeAOnly()` from `ListCompareGuiTool`.

Reply:
Yes, that's right, my bad, at beginning, i used those properties